### DISCUSSION TITLE
GAMECUBE SWISS Question?

### DISCUSSION THREAD

**Discussion by FRANKENSTEIN888** [emukidid/swiss-gc#668] [General]
Are there any emulators that work on Nintendo Game Cube with Swiss??? N64,NES,SNES,SEGA Genesis, GBA, others???

**Comment by Extrems** [comment]
Dude, you were already given a comprehensive answer.

**Reply by FRANKENSTEIN888** [reply]
Dude I am 50 Years old so stop calling me DUDE Jack Ass! Also I never got any message about any reply and my Post was Deleted DUDE!

**Reply by Extrems** [reply]
#667
